[Feat] PlaceView, PointView API 연동

jaminleee opened the pull request:
## 🔥*Pull requests*

⛳️ **작업한 브랜치**
- #48

👷 **작업한 내용**
<!-- 작업한 내용을 적어주세요. -->
- PlaceView
- PointView API 연동

## 🚨 참고 사항
<!-- 참고할 사항이 있다면 적어주세요. -->
> 내가 참고할 사항..
- AppData, DTO 명확하게 구분하여 사용하기⭐️⭐️⭐️⭐️⭐️

## 📸 스크린샷
|기능|스크린샷|
|:--:|:--:|
|GIF|<img src = "https://github.com/SOPT-33RD-APP-NAVERPAY/NaverPay-iOS/assets/91969458/a7cf649d-aab7-4be1-9fa2-86098dbff97c" width ="250">|

## 📟 관련 이슈
- Resolved: #48

jaminleee committed:
[Feat] PlaceView API 연동

jaminleee committed:
[Feat] PointView API 연동

sjk4618 reviewed (approved):
수고하셨습니다!!

kim-seonwoo reviewed (commented):
네트워크 연결하느라 수고하셨어요~ ㅠ!

jaminleee committed:
Merge branch 'main' into #48

jaminleee committed:
[Fix] Conflict 해결 (#48)

# Conflicts:
#	NaverPay.xcodeproj/project.pbxproj
#	NaverPay/Application/SceneDelegate.swift

jaminleee committed:
[Fix] Conflict 해결

# Conflicts:
#	NaverPay/Application/SceneDelegate.swift

jaminleee committed:
[Chore] 코드 수정
